renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -174,6 +176,115 @@ class EGLHook : LibraryHook, public GLPlatform
       eglDestroyContext_real(context.egl_dpy, context.egl_ctx);
   }
 
+  void DeleteReplayContext(GLWindowingData context)
+  {
+    if(eglDestroyContext_real)
+    {
+      eglMakeCurrent_real(context.egl_dpy, 0L, 0L, NULL);
+      eglDestroyContext_real(context.egl_dpy, context.egl_ctx);
+    }
+  }
+
+  void SwapBuffers(GLWindowingData context)
+  {
+    eglSwapBuffers_real(context.egl_dpy, context.egl_wnd);
+  }
+
+  void GetOutputWindowDimensions(GLWindowingData context, int32_t &w, int32_t &h)
+  {
+    eglQuerySurface_real(context.egl_dpy, context.egl_wnd, EGL_WIDTH, &w);
+    eglQuerySurface_real(context.egl_dpy, context.egl_wnd, EGL_HEIGHT, &h);
+  }
+
+  bool IsOutputWindowVisible(GLWindowingData context)
+  {
+    GLNOTIMP("Optimisation missing - output window always returning true");
```

Comment:
Given that EGL is primarily going to be used on Android (where we won't even be creating output windows), we can probably just make this function return true silently.